apps/frontend/src/components/ui/dashboard/RevenueTransaction.vue

```diff
@@ -104,17 +104,25 @@ function formatTransactionStatus(status: string): string {
 	return capitalizeString(status)
 }
 
-function formatMethodName(method: string | undefined): string {
+function formatMethodName(method: string | undefined, method_id: string | undefined): string {
 	if (!method) return 'Unknown'
 	switch (method) {
 		case 'paypal':
 			return 'PayPal'
 		case 'venmo':
 			return 'Venmo'
 		case 'tremendous':
-			return 'Tremendous'
+			if (method_id === 'O7VZ5WQOCUQM') {
```

Comment:
This is good enough for now - but i will make a PR tomo to add a cache of all tremendous methods to the generated state/tags stuff so we can be accurate here